pubspec.yaml

```diff
@@ -16,7 +16,7 @@ dependencies:
   flutter:
     sdk: flutter
   flutter_layout_grid: ^2.0.0
-  intl: ^0.17.0
+  intl: ">=0.17.0 <1.0.0"
```

Comment:
Future updates to [<kbd>intl</kbd>](https://pub.dev/packages/intl) might contain breaking changes, hence we can only update to `^0.18.0` and not allow just any version: https://dart.dev/tools/pub/versioning#semantic-versions.

```suggestion
  intl: ^0.18.0
```

But even then, we're still blocked by https://github.com/xantiagoma/dart_date/issues/26

Reply:
yeah((